garrysmod/gamemodes/terrortown/gamemode/player_ext.lua

```diff
@@ -52,13 +52,13 @@ function plymeta:SubtractCredits(amt) self:AddCredits(-amt) end
 
 function plymeta:SetDefaultCredits()
    if self:GetTraitor() then
-      local c = GetConVarNumber("ttt_credits_starting")
+      local c = GetConVar("ttt_credits_starting"):GetInt()

```

Comment:
FYI, this does not achieve anything. This is exactly what `GetConVarNumber` does behind the scenes.

Reply:
Just means there's less deprecated shit for glualint to blind me with is all.
Also, I wanted to cache the result of GetConVar outside of each function, but it caused errors in the area you've highlighted so I left it as is in case it caused anything else to break.